pages/showcase.md

```diff
@@ -32,24 +32,24 @@ If you want your application to be featured here submit a PR to the [website](ht
     <div class="col-xs-12 col-sm-6 col-md-4">
         <div class="thumbnail no-margin-bottom">
             <div class="video-container">
-                <img width="560" height="315" src="{{ site.url }}/images/showcase/beta.devoxx.com.png" />
```

Comment:
Can you keep devoxx.com. the correct URL is devoxx.com

Reply:
@deepu105 : Thanks. Done. :+1: